content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,170 @@
+for any special instructions for your particular environment.
+
+## Initial preparations
+
+You need to do a few one-time steps in order to be able to setup a multicluster mesh:
+
+1. Ensure that all of your clusters are included in your [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
+and create contexts for each cluster. Once you're done, your configuration file should include something similar to:
+
+    {{< text syntax="yaml" downloadas="kubeconfig.yaml" >}}
+    kind: Config
+    apiVersion: v1
+    clusters:
+    - cluster:
+      name: cluster-east-1
+    - cluster:
+      name: cluster-east-2
+    - cluster:
+      name: cluster-west-1
+    contexts:
+    - context:
+        cluster: cluster-east-1
+      name: context-east-1
+    - context:
+        cluster: cluster-east-2
+      name: context-east-2
+    - context:
+        cluster: cluster-west-1
+      name: context-west-1
+    {{< /text >}}    
+
+1. Decide on what the name of your multicluster mesh will be. Something short but memorable is your best choice here:
+
+    {{< text bash >}}
+    $ export MESH_ID=mymeshname
+    {{< /text >}}
+
+1. Decide on the organization name to use in the root and intermediate certificates created to let the clusters communicate with one
+another. This should generally be derived from your organization's DNS name:
+
+    {{< text bash >}}
+    $ export ORG_NAME=mymeshname.mycompanyname.com
+    {{< /text >}}
+
+1. Create a working directory where to store a number of files produced during the cluster
+onboarding process:
+
+    {{< text bash >}}
+    $ export WORKDIR=mydir
+    $ mkdir -p ${WORKDIR}
+    $ cd ${WORKDIR}
+    {{< /text >}}
+
+1. Download the setup script to your working directory. This script takes care of creating the requisite
+certificates to enable cross-cluster communication, it prepares default configuration files for you,
+and will deploy and configure Istio in each cluster. 
+
+    {{< text bash >}}
+    $ curl @{{< github_file >}}/samples/multicluster/setup-mesh@ >./setup-mesh.sh
+    $ chmod +x ./setup-mesh.sh
+    {{< /text >}}
+
+1. And finally, prepare the mesh. This will create a root key and certificate that will
+be used to secure communication between the clusters in the mesh, along with a `base.yaml`
+file which will be used to control the Istio configuration deployed on all the cluaters:
+
+    {{< text bash >}}
+    $ ./setup-mesh.sh prep
+    {{< /text >}}
+    
+{{< warning >}}
+The final step above created some secret material in your working directory, in particular some private keys associated
+with the certificate. You should store and protect those secrets. The specific files to safeguard are:
+
+{{< text plain >}}
+certs/root-key.pem - the root's private key.
+certs/intermediate-*/ca-key.pem - intermediates' private keys
+{{< /text >}}
```

Comment:
The intermediate keys are generated further down during the `apply` step. But the same warning about protecting them applies.

Reply:
OK, I moved the section lower.